### DISCUSSION TITLE
Update ps_facetedsearch dependency to last version on 8.1.x

### DISCUSSION THREAD

**Discussion by lartist** [PrestaShop/PrestaShop#31280] [Ideas]
### The problem

Given for releases we try to keep prestashop module dependencies updated. 
And to build a release for the 8.1.x the update of ps_facetedsearch causes problems (auto tests fail).
Then we decided to release 8.1.x (at least the beta) without updating ps_facetedsearch dependency (meaning it stays at v3.10.0 in composer.lock).
And this update will be handle after the beta building.
 

### The solution

The auto tests should probably be changed.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
Hi @lartist 

I think we can discuss it with other committers—no need to create a discussion about it there :)

**Reply by lartist** [reply]
Hi @kpodemski actually @matks asked me to create a feature request for this topic and when you click en feature request it creates a discussion. I don't know why !

**Reply by kpodemski** [reply]
That's right. We moved feature requests to discussions. I'm unsure what we could discuss regarding the faceted search. It looks like either an update on the release status in the Issue or information to #project-members on Slack 👍🏻 All good. Thanks @lartist .

**Comment by Hlavtox** [comment]
@lartist Faceted search is already fixed, just update the dependency to 3.11.1

**Reply by lartist** [reply]
I'll test it for the next release (beta-2 or RC-1). Thanks @Hlavtox

**Reply by lartist** [reply]
An attempt already failed https://github.com/PrestaShop/PrestaShop/pull/31266 . @Hlavtox 
do you any idea how to update faceted search without downgrade another dependency ?

**Reply by Hlavtox** [reply]
@lartist I don't understand what you want from me :D

In https://github.com/PrestaShop/PrestaShop/pull/31266 I see that the tests failed because of BO order, not because of the module.

What dependency downgrade you are talking about?

**Reply by lartist** [reply]
@Hlavtox https://github.com/PrestaShop/PrestaShop/pull/31266/files#diff-f37acfaa6b11f575a9a6f41a75fa73a61d0f8ebc2c9b8cddc215d8aca10e44f5R13599

**Reply by Hlavtox** [reply]
@kpodemski Can you please explain what he means?

**Reply by kpodemski** [reply]
@lartist it was fixed there:

https://github.com/PrestaShop/PrestaShop/pull/31518/files
